pkg/rc/farm.go

```diff
@@ -277,12 +278,17 @@ func (rcf *Farm) failsafe(rcFields []rcstore.RCLockResult) {
 		}
 		panic("No RCs are scheduled at all. Create one RC to enable the farm. Panicking to escape a potentially bad situation.")
 	}
+
 	globalReplicaCount := 0
 	for _, rc := range rcFields {
-		if rc.ReplicasDesired > 0 {
+		if rc.ReplicasDesired == nil {
+			panic(fmt.Sprintf("RC %s is missing replicas_desired count", rc.ID))
```

Comment:
I was thinking that if one RC is missing `replicas_desired` then probably all of them are corrupted (e.g. a code change changed the json key). In that case we'd want the server to bail as soon as possible.

I think for now i'd like to keep this heavy handed approach and then if it poses a problem we can trust the unmarshaling errors